Week 2 - Arrays/substitution/substitution.c

```diff
@@ -0,0 +1,95 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+
+bool isValid(string key);
+int getIndex(char letter);
+
+int SIZE = 26;
+char ALPHABET[26] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
+                     'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'};
+int values[26];
+
+int main(int argc, string argv[])
+{
+    if (argc != 2)
+    {
+        printf("Usage ./substitution key.\n");
+        return 1;
+    }
+
+    string key = argv[1];
+
+    if (strlen(key) != 26)
+    {
+        printf("Key must contain 26 characters.\n");
+        return 1;
+    }
+
+    if (!isValid(key))
+    {
+        printf("Key should contaim each letter exactly once.\n");
+        return 1;
+    }
+
+    string text = get_string("plaintext:  ");
+
+    int textSize = strlen(text);
+    char cipher[textSize];
```

Comment:
Not clear at all when I can use strings in C and when not.. I am just trying to use samples from the lecture assuming it is an effective way to do it. Otherwise what's the point of the lecture 🤷‍♀️

Will try to play around with strings, right.

Reply:
Please share on Monday what struggles you have with strings, I will try to do my best to help with them and explain any difficulties 🙏🏾 Whenever you include cs50 library you can safely use strings as variables as well as return values.